src/rebar_cmd_prv.erl

```diff
@@ -49,36 +59,78 @@ format_error(Reason) ->
 %% ===================================================================
 %% Private API
 %% ===================================================================
-run_shell({CmdName, Command}) ->
-    run_shell({CmdName, Command}, 15000).
 
-run_shell({CmdName, Command}, Timeout) ->
-    Port = erlang:open_port({spawn, Command}, [exit_status]),
```

Comment:
As discussed elsewhere, you remove the ability to obtain the exit status. Might not be a problem, though. You tell me.

Reply:
Yeah I think this is fine, as long as we can report a failure or a success.